packages/amplify-e2e-core/src/categories/lambda-layer.ts

```diff
@@ -132,4 +154,31 @@ function getLayerRuntimeInfo(runtime: LayerRuntimes) {
     default:
       throw new Error(`Invalid runtime value: ${runtime}`);
   }
+};
+
+function printFlow(chain : ExecutionContext , settings : any ,layerDirRegex , runtimeDisplayNames){
```

Comment:
you don't need to wait on every line. It would be sufficient to just wait on the last 2 lines which would eliminate a lot of that regex stuff

Reply:
It's there for validation. If someone makes a text change by accident, the tests will catch it.

Reply:
If you want to check all the lines except that regex one that would be fine. But that regex is a lot of complexity for marginal test benefit in my opinion. It could also easily be a small unit test around the printout of the method rather than in the e2e tests